pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -122,12 +123,15 @@ type VSphereConfig struct {
 		WorkingDir string `gcfg:"working-dir"`
 		// Soap round tripper count (retries = RoundTripper - 1)
 		RoundTripperCount uint `gcfg:"soap-roundtrip-count"`
+		// VMUUID is the virtual machine's UUID. If not set, will be fetched from the machine.
```

Comment:
We have many UUIDs! How about following;

// VM Instance UUID of virtual machine which can be retrieved from instanceUuid property in VmConfigInfo, or also set as vc.uuid in VMX file) ?